packages/postgres-database/src/simcore_postgres_database/models/resource_tracker_wallets_credit_transactions.py

```diff
@@ -0,0 +1,98 @@
+""" Wallets credit transaction
+    - Basically this table is balance sheet of each wallet.
+"""
+import enum
+
+import sqlalchemy as sa
+
+from ._common import column_modified_datetime
+from .base import metadata
+
+
+class TransactionBillingStatus(str, enum.Enum):
+    PENDING = "PENDING"
+    BILLED = "BILLED"
+    NOT_BILLED = "NOT_BILLED"
+    MANUAL_REVIEW = "MANUAL_REVIEW"
```

Comment:
suggestion: maybe something like this?
```suggestion
    REQUIRES_MANUAL_REVIEW = "REQUIRES_MANUAL_REVIEW"
```

Reply:
Thanks, I like it, i will rename :+1: